builders/xcm/fees.md

```diff
@@ -0,0 +1,230 @@
+
+Although Polkadot doesn't currently use database weight units to calculate costs, the weight units for database operations are shared here for reference.
+
+|     Database      |    Read    |   Write    |
+|:-----------------:|:----------:|:----------:|
+| RocksDB (default) | 20,499,000 | 83,471,000 |
+|     ParityDB      | 11,826,000 | 30,052,000 |
+
+With the instruction weight cost established, you can calculate the cost of the instruction in DOT. 
+
+In Polkadot, the `ExtrinsicBaseWeight` is set to `85,212,000` which is mapped to 1/10 cents. Where 1 cent is `10^10 / 10,000`. Therefore, the formula for calculating the final fee in DOT looks like this:
+
+```
+Total-Planck-DOT = TotalWeight * PlanckDOT * (1 / ExtrinsicBaseWeight)
+DOT = Total-Planck-DOT / 10^10
+```
+
+Therefore, the actual calculation for the `WithdrawAsset` instruction is:
+
+```
+Total-Planck-DOT = 1000000000 * (10^10 / 1000) * (1 / 85212000)
+DOT = 117354360 / 10^10
+```
+
+The total cost is `0.0117354363 DOT`.
+
+### Kusama {: #kusama }
+
+Kusama doesn't used fixed weights, but bases the weights off of measured results from benchmarking data. The weights on Kusama take into consideration database reads and writes in addition to the weight required for a given instruction. 
+
+The breakdown of weight costs for database read and write operations are as follows:
+
+|     Database      |    Read    |    Write    |
+|:-----------------:|:----------:|:-----------:|
+| RocksDB (default) | 25,000,000 | 100,000,000 |
+|     ParityDB      | 8,000,000  | 50,000,000  |
+
+Now that you are aware of the weight costs for database reads and writes on Kusama, you can calculate the weight cost for a given instruction using the base weight for an instruction. 
+
+The `WithdrawAsset` instruction has a base weight of `20,385,000`, and includes one database read, and one database write. Therefore, the total weight cost of the `WithdrawAsset` instruction is calculated like so:
+
+```
+20,385,000 + 25,000,000 + 100,000,000 = 145,385,000
+```
+
+The `BuyExecution` instruction has a base weight of `3,109,000` and doesn't include any database reads or writes. Therefore, the total weight cost of the `BuyExecution` instruction is `3,109,000`.
+
+With the instruction weight cost established, you can calculate the cost of the instruction in KSM. 
+
+In Kusama, the `ExtrinsicBaseWeight` is set to `86,309,000` which is mapped to 1/10 cents. Where 1 cent is `10^12 / 30,000`. Therefore, the formula for calculating the final fee in KSM looks like this:
+
+```
+Total-Planck-KSM = TotalWeight * PlanckKSM * (1 / ExtrinsicBaseWeight)
+KSM = Total-Planck-KSM / 10^12
+```
+
+Therefore, the actual calculation for the `WithdrawAsset` instruction is:
+
+```
+Total-Planck-KSM = 145,385,000 * (10^12 / 30000 * 10) * (1 / 86309000)
+KSM = 5614903.04216 / 10^12
+```
+
+The total cost is `0.00000561490304213694 KSM`.
```

Comment:
Add an example of the weights and KSM cost of a XCM Token transfer to Kusama (same as we did in the learning session). You can probably do this in a table?